Classes/Issues/Comments/Details/IssueCommentDetailCell.swift

```diff
@@ -127,19 +147,25 @@ final class IssueCommentDetailCell: IssueCommentBaseCell, ListBindable {
         dateLabel.setText(date: viewModel.date)
         loginLabel.text = viewModel.login
 
+        let editedString: String
+
         if let editedLogin = viewModel.editedBy, let editedDate = viewModel.editedAt {
             editedLabel.isHidden = false
 
             let editedByNonOwner = NSLocalizedString("Edited by %@", comment: "")
             let editedByOwner = NSLocalizedString("Edited", comment: "")
             let format = viewModel.login != editedLogin ? editedByNonOwner : editedByOwner
-            editedLabel.text = "\(Constants.Strings.bullet) \(String(format: format, editedLogin))"
+            editedString = String(format: format, editedLogin)
+            editedLabel.text = "\(Constants.Strings.bullet) \(editedString)"
 
             let detailFormat = NSLocalizedString("%@ edited this issue %@", comment: "")
             editedLabel.detailText = String(format: detailFormat, arguments: [editedLogin, editedDate.agoString])
         } else {
             editedLabel.isHidden = true
+            editedString = ""
         }
+
+        accessibilityLabel = NSLocalizedString("\(viewModel.login) commented \(viewModel.date.agoString). \(editedString)", comment: "")
```

Comment:
Not sure using a localized string is best here, there's no way we could translate every possible mutation of this -- Maybe if we use `String(format:` where the format is localizable?

Reply:
Oops, of course.